package-lock.json

```diff
@@ -8342,4 +8342,4 @@
       "dev": true
     }
   }
-}
```

Comment:
it is strange that you added new dependency, but this is the only change in package-lock 🤔